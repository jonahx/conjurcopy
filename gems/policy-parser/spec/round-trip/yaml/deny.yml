- !deny
  privilege: read
  resource: !variable foobar
  role: !group developers

- !deny
  privilege: [ read, execute ]
  resource: !variable baz
  role: !group developers

- !deny
  privilege: read
  resource: !variable db_password
  role: !group
    id: developers

- !deny
  privilege: read
  resource: !variable mongo_password
  role: !group developers
