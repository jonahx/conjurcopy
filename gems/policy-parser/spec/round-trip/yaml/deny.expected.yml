---
- !deny
  privilege: read
  resource: !variable
    id: foobar
  role: !group
    id: developers
- !deny
  privilege:
  - read
  - execute
  resource: !variable
    id: baz
  role: !group
    id: developers
- !deny
  privilege: read
  resource: !variable
    id: db_password
  role: !group
    id: developers
- !deny
  privilege: read
  resource: !variable
    id: mongo_password
  role: !group
    id: developers
